amplify/__tests__/functions/send-email/handler.test.ts

```diff
@@ -18,7 +18,7 @@ type HandlerCallbackType = Parameters<
 beforeAll(() => {
   jest.useFakeTimers();
   jest.setSystemTime(new Date('2022-01-01 10:00'));
-  process.env.NOTIFY_DOMAIN_NAME = 'test.notify.nhs.uk'
+  process.env.NOTIFY_DOMAIN_NAME = 'test.notify.nhs.uk';
```

Comment:
i think it's good practice to unset test environment variables at the end of tests when you don't need them anymore - it just ensures they don't mess with other tests (may not affect anything now but could cause hard to diagnose issues in future)

Reply:
Yeah that's a fair shout. I've reset the envVar back to what it was in an afterAll if that's what you meant by the unset.